src/overlays/actors/ovl_Arrow_Fire/z_arrow_fire.c

```diff
@@ -212,10 +215,10 @@ void ArrowFire_Draw(Actor* thisx, GlobalContext* globalCtx2) {
         Matrix_Scale(0.01f, 0.01f, 0.01f, MTXMODE_APPLY);
 
         // Draw red effect over the screen when arrow hits
-        if (this->unk_15C > 0) {
+        if (this->hitAnimIntensity > 0) {
```

Comment:
Usually we use "anim" when there's an animated mesh involved (typically using the SkelAnime system)

It is very common that values change every frame (so are animated), so we just don't include that information in the name usually. (as an example among many, the `BgGanonOtyuka` struct has members which change every frame so are "animated", but none of them mentions it)

So at least I would remove "anim" so `hitIntensity`